src/ansys/geometry/core/designer/edge.py

```diff
@@ -142,7 +143,13 @@ def shape(self) -> TrimmedCurve:
     @ensure_design_is_active
     def length(self) -> Quantity:
```

Comment:
Is the decorator needed here too?

Reply:
No, there is a try except inside. If it fails due to the backend version error, we fallback to the old implementation.